easy/12.js

```diff
@@ -1,5 +1,8 @@
 function convertToDecimal(perc) {
-	
+    const arr = perc.map((n) => {

```

Comment:
You can remove unnecessary variable, skip curly brackets and return

Reply:
how to skip curly brackets?